dataactvalidator/filestreaming/fieldCleaner.py

```diff
@@ -146,10 +147,11 @@ def padField(field,value,fileType,validationInterface):
         if value is not None and validationInterface.isPadded(field,fileType):
             # If padded flag is true, get column length
             padLength = validationInterface.getColumnLength(field, fileType)
-            # Pad to specified length with leading zeros
-            return value.zfill(padLength)
-        else:
-            return value
+            if len(value) < padLength:
```

Comment:
Just checking my assumptions here...I _think_ `zfill` will just ignore your padding request if the length of the field in question is already >= the length you're padding to. Is there a case that requires us to have this explicit `if len(value) < padLength` check?